deployment/terraform/variables.tf

```diff
@@ -219,3 +219,63 @@ variable "rds_cpu_credit_balance_threshold" {
   default = 30
   type    = number
 }
+
+variable "batch_cpu_container_instance_volume_size" {
+  type    = number
+  default = 30
+}
+
+variable "batch_gpu_container_instance_volume_size" {
+  type    = number
+  default = 30
```

Comment:
I think that we may want this to default to a larger number (e.g., 128GB) because scratch space used by the jobs will reuse this space.